airbyte-integrations/connectors/source-box-data-extract/metadata.yaml

```diff
@@ -0,0 +1,39 @@
+data:
+  ab_internal:
+    ql: 300
+    sl: 300
+  allowedHosts:
+    hosts:
+      - box.com
+      - boxcloud.com
+  # registries:
```

Comment:
Note to self: our template has invalid keys in it. This should be `registryOverrides`, but this should be actually uncommented, and enabling for both OSS and Cloud is fine.

Reply:
Ah!, Airbyte-CI was complaining about that, and since I have no idea of what it does I just comment it out.
I could use your advice on what to put in these:
```
ab_internal:
    ql: 300
    sl: 300
```

Reply:
fixed the "registryOverrides", bumped base to 4.0.0